lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -3,6 +3,7 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
+// @ts-nocheck
```

Comment:
is this needed?

Reply:
yeah. The file reaches into `WebInspector.CSSMatchedStyles` and mutates types in filter/map pipelines which always ends up as a lot of work in TS. I got pretty far but eventually gave up, so settled just for param/return types for now :)

Reply:
ah ok, sg